mdn/developer-portal
# 526: Remove the hard-coded locale from the link to Firefox

This changeset addresses the problem of the link to Firefox in the navigation pointing to 
the en-GB locale by default.

(Resolves #526 )

## Key changes:

- Remove the hard-coded locale from the link to Firefox. https://www.mozilla.org/firefox/ will automatically set a locale based on user preferences.


## How to test 

This code isn't on staging yet, so:
- Pleas ereview the changes in the "1 file changed" tab
- check, yourself that https://www.mozilla.org/firefox/ will add the relevant locale to the URL after you follow it
